services/gateway/gRPC_client.py

```diff
@@ -0,0 +1,20 @@
+import grpc
+import user_pb2
+import user_pb2_grpc
+
+class GRPCClient(object):
+    def __init__(self):
+        self.host = '127.0.0.1'
+        self.server_port = '8000'
+
+        self.channel = grpc.insecure_channel('{}:{}'.format(self.host, self.server_port))
```

Comment:
Would these be equivalent? We should define one form of formatting strings and stick with it across the code base.
```suggestion
        self.channel = grpc.insecure_channel(f'{self.host}:{self.server_port}')
```

Reply:
Honest question here.

Reply:
This file will not be used